sockeye/arguments.py

```diff
@@ -249,7 +249,15 @@ def add_model_parameters(params):
 
     model_params.add_argument('--weight-tying',
```

Comment:
To me 'tying' has a weaker implication (e.g. it could mean that the parameters should be made 'similar'), whereas 'sharing' clearly says that these are the same set of parameters.
but if you feel strongly about backwards compatibility we can leave as is.